From: Kill Your TV <killyourtv@i2pmail.org>
Date: Wed, 17 Dec 2014 15:30:52 +0000
Subject: fix package build

---
 Makefile | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index 7e512cb..6646aed 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,5 @@
 UNAME := $(shell uname -s)
+GREP := fgrep
 SHLIB := libi2pd.so
 I2PD  := i2p
 
@@ -10,7 +11,7 @@ USE_STATIC := no
 ifeq ($(UNAME),Darwin)
   DAEMON_SRC += DaemonLinux.cpp
 	include Makefile.osx
-else ifeq ($(UNAME),FreeBSD)
+else ifeq ($(shell echo $(UNAME) | $(GREP) -c FreeBSD),1)
   DAEMON_SRC += DaemonLinux.cpp
 	include Makefile.bsd
 else ifeq ($(UNAME),Linux)
@@ -26,17 +27,17 @@ obj:
 	mkdir -p obj
 
 obj/%.o : %.cpp %.h
-	$(CXX) $(CXXFLAGS) $(INCFLAGS) -c -o $@ $<
+	$(CXX) $(CXXFLAGS) $(shell dpkg-buildflags --get CXXFLAGS) $(INCFLAGS) -c -o $@ $<
 
 # weaker rule for building files without headers
 obj/%.o : %.cpp
-	$(CXX) $(CXXFLAGS) $(INCFLAGS) -c -o $@ $<
+	$(CXX) $(CXXFLAGS) $(shell dpkg-buildflags --get CXXFLAGS) $(INCFLAGS) -c -o $@ $<
 
 $(I2PD):  $(patsubst %.cpp,obj/%.o,$(DAEMON_SRC))
-	$(CXX) -o $@ $^ $(LDFLAGS)  $(LDLIBS)
+	$(CXX) -o $@ $^ $(LDFLAGS)  $(shell dpkg-buildflags --get LDFLAGS) $(LDLIBS)
 
 $(SHLIB): $(patsubst %.cpp,obj/%.o,$(LIB_SRC))
-	$(CXX) -o  $@ $^ $(LDFLAGS)  $(LDLIBS) -shared 
+	$(CXX) -o  $@ $^ $(LDFLAGS)  $(shell dpkg-buildflags --get LDFLAGS) $(LDLIBS) -shared
 
 clean:
 	rm -fr obj $(I2PD) $(SHLIB)
