From: Kill Your TV <killyourtv@i2pmail.org>
Date: Wed, 4 Feb 2015 14:44:53 +0000
Subject: old boost FTBFS

---
 Transports.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Transports.cpp b/Transports.cpp
index 2f0c95c..ddfde44 100644
--- a/Transports.cpp
+++ b/Transports.cpp
@@ -255,7 +255,13 @@ namespace transport
 				auto address = peer.router->GetNTCPAddress (!context.SupportsV6 ());
 				if (address)
 				{
+#if BOOST_VERSION >= 104900
 					if (!address->host.is_unspecified ()) // we have address now
+#else
+					boost::system::error_code ecode;
+					address->host.to_string (ecode);
+					if (!ecode)
+#endif
 					{
 						if (!peer.router->UsesIntroducer () && !peer.router->IsUnreachable ())
 						{	
