From 0b77c9651b11ff155707514bbd6989654d9bb2d2 Mon Sep 17 00:00:00 2001
From: orignal <i2porignal@yandex.ru>
Date: Thu, 30 Oct 2014 18:04:17 -0400
Subject: [PATCH 3/3] 0.2.0

---
 version.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/version.h b/version.h
index 35e092e..b12cb89 100644
--- a/version.h
+++ b/version.h
@@ -2,7 +2,7 @@
 #define _VERSION_H_
 
 #define CODENAME "Purple"
-#define VERSION "0.1.0"
+#define VERSION "0.2.0"
 #define I2P_VERSION "0.9.15"
 
 #endif
-- 
2.1.1

